{{cookiecutter.project_slug}}/Makefile

```diff
@@ -62,8 +62,8 @@ clean-pyc: ## remove Python file artifacts
 lint: ## check style with flake8
 	flake8 {{ cookiecutter.python_package }}
 
-test: ## run anyblok nose tests
-	anyblok_nose -c app.test.cfg -- -v -s {{ cookiecutter.python_package }}
+test: ## run tests
+	ANYBLOK_DATABASE_NAME={{ cookiecutter.db_name }}_test ANYBLOK_DATABASE_DRIVER={{ cookiecutter.db_driver_name }} py.test -v -s {{ cookiecutter.python_package }}
```

Comment:
adapt according the outcome of https://github.com/AnyBlok/AnyBlok/pull/101#discussion_r296000467 or remove ``app.test.cfg``